### ISSUE TITLE
iOS (KMM) & Flutter support

### ISSUE THREAD

**Issue by b95505017** [algolia/algoliasearch-client-kotlin#306]
https://www.algolia.com/doc/guides/building-search-ui/getting-started/how-to/flutter/android/
This flutter codelab only target Android, but it seems that all dependencies are based on `kotlin multiplatform`. Is it possible to use same idea to support flutter iOS app with this kotlin library?

**Comment by Aallam** [comment]
Hey! you can find the iOS counterpart [here](https://www.algolia.com/doc/guides/building-search-ui/getting-started/how-to/flutter/ios/). For the multiplatform support, we only target JVM at the moment, but this might change in the future.

**Comment by b95505017** [comment]
@Aallam Yes I know this page, but currently we want to use same codebase with Algolia in Android/iOS (Flutter + kotlin multiplatform). It there any way to achieve this?

**Comment by Aallam** [comment]
Currently, the Kotlin client does not support KMM. It is in our backlog, but we can't confirm a release date.

**Comment by julien-h-webcenter** [comment]
i'm in the same but not with flutter. Only KMM + jetpack compose + swiftUI and i tried to put the Kotlin API Client for this projet. but i undestand that i can't to use it. It could to be nice to support KMM :)

**Comment by b95505017** [comment]
@Aallam Any chance to merge it?

**Comment by b95505017** [comment]
@Aallam Why close #310 ?
